### ISSUE TITLE
Duplicate Babel register for tests?

### ISSUE THREAD

**Issue by sprjr** [babel/generator-babel-boilerplate#428]
Heya,

While looking through the `gulefile.js` and `node.js` files, I noticed that it appears to be registering babel twice when running the mocha tests:

It occurs [here when setting up node](https://github.com/babel/generator-babel-boilerplate/blob/master/app/templates/test/setup/node.js#L5):, which is imported for the `_mocha()` invocation [here](https://github.com/babel/generator-babel-boilerplate/blob/master/app/templates/gulpfile.js#L82)

It is _also_ done [here](https://github.com/babel/generator-babel-boilerplate/blob/master/app/templates/gulpfile.js#L95) prior to calling `_mocha()`.

Seems like a dupe, should it be removed from one of the places?

**Comment by jmeas** [comment]
Good catch! I believe the require in the node setup can be removed. It must be leftover from a previous iteration.

I'm planning the next release of this generator – I'll be sure to sort this out then.
